roles/variables/defaults/main.yml

```diff
@@ -1463,6 +1463,9 @@ sso_groups_scope: none
 ### Configures whether offline_access scope would be requested in the authorization URI, Set this to false if offline tokens are not allowed for the user or client in IdP
 sso_refresh_token: true
 
+### SSL certificate (full path of file on control node) of IDP Domain for SSO in C3/cli. Optional, needed when IDP server has TLS enabled
+sso_idp_pem_path: ""
```

Comment:
I think there should be one more option to confirm if we need to import the certs. In production, most of the time, the CA root will be available in the JVM. 
So import only when "import_idp_certs" is true and "sso_idp_pem_path" is empty.

Reply:
I think you meant `only when "import_idp_certs" is true and "sso_idp_pem_path" is not empty.?` But the condition I have used to import is when  "sso_idp_pem_path" is not empty. There is no need to provide this path otherwise, right?

Reply:
Understand. User is expected to have the certs imported to file in order to get it configured with services.